code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1730,9 +1738,9 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/medicine/ondansetron/on_mob_life(mob/living/carbon/M, seconds_per_tick, times_fired)
-	. = ..()
 	if(SPT_PROB(8, seconds_per_tick))
-		M.adjust_drowsiness(2 SECONDS * REM * seconds_per_tick)
+		. = ..()
```

Comment:
I think this one is also wrong, we now _only_ metabolise the reagent 8% of the time, which I don't think is how it used to work?
Should probably be
```
. = ..()
if(SPT_PROB(8, seconds_per_tick))
	M.adjust_drowsiness(2 SECONDS * REM * seconds_per_tick)
```
as before

Reply:
Not sure how the drowsiness effect was lost.